scripts/build_esri_projection_mapping.py

```diff
@@ -459,9 +459,19 @@
         - Longitude_Of_Center: EPSG_NAME_PARAMETER_LONGITUDE_OF_NATURAL_ORIGIN
         - Latitude_Of_Center: EPSG_NAME_PARAMETER_LATITUDE_OF_NATURAL_ORIGIN
 
-# Times: not handled, but present in PROJ
+- Times:
+    WKT2_name: PROJ_WKT2_NAME_METHOD_TIMES
+    Params:
+        - False_Easting: EPSG_NAME_PARAMETER_FALSE_EASTING
+        - False_Northing: EPSG_NAME_PARAMETER_FALSE_NORTHING
+        - Central_Meridian: EPSG_NAME_PARAMETER_LONGITUDE_OF_NATURAL_ORIGIN
 
-# Vertical_Near_Side_Perspective: not handled, but present in PROJ
+- Vertical_Near_Side_Perspective:
+    WKT2_name: PROJ_WKT2_NAME_METHOD_VERTICAL_NEAR_SIDE_PERSPECTIVE
+    Params:
```

Comment:
missing Height and Latitude_Of_Center parameters. See above remark.

Reply:
I'll fix, thanks